test/config copy.ini

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2013-2016 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+# These environment variables are set by the build process and read by
+# test/functional/test_runner.py and test/util/tapyrus-util-test.py
+
+[environment]
+SRCDIR=/Users/navia/Documents/Projects/Tapyrus/tapyrus-core-nb/FunctionalTestTimeout
+BUILDDIR=/Users/navia/Documents/Projects/Tapyrus/tapyrus-core-nb/FunctionalTestTimeout/build
+EXEEXT=
+RPCAUTH=/Users/navia/Documents/Projects/Tapyrus/tapyrus-core-nb/FunctionalTestTimeout/share/rpcauth/rpcauth.py
```

Comment:
Individual environmental information is undesirable.

Reply:
this file was added by accident. it shouldn't be there